src/cow-react/pages/Swap/SwapMod.tsx

```diff
@@ -367,6 +368,8 @@ export default function Swap({ history, location, className }: RouteComponentPro
                     {!isExpertMode && !allowedSlippage.equalTo(INITIAL_ALLOWED_SLIPPAGE_PERCENT) && (
                       <RowSlippage allowedSlippage={allowedSlippage} />
                     )}
+                    {!isExpertMode && <RowDeadline />}
```

Comment:
Are you adding this to the new swap too?

Reply:
it's already there inside the `TradeSUmmary` or `TradeBasicDetails` one of those, or both

Reply:
nope wasn't u right, added